### ISSUE TITLE
Searches not finding any results

### ISSUE THREAD

**Issue by soulvoid86** [CouchPotato/CouchPotatoServer#7120]
### Steps to reproduce:
New user to Couch Potato/Sonarr. Have never used any system like this until today. I've used Plex for a very long time and am trying to integrate this through PlexRequests.net, but I'm stuck at the very beginning stage. I simply can't get Couch Potato to find or download anything through IPT/Bittorrent. (I've got all other steps up to this point working fine, not looking for help on anything other than getting CP to search and download! :))

I've used IPTorrents 99.9% of the time up to today. I was able to get CP to download  1 item from kickasstorrents one time, but it was a dead torrent anyway. After that, it never worked again. I've ran through a lot of the basics, reinstall CP, clearing out all folders and starting fresh, it's not being block anywhere on the network, no other service is running on the ports, etc.

I've also tried using uTorrent, Hadouken and Deluge, with no success. (I'd like to stick to BitTorrent, it's what I've been using for a while.) CP connects to it fine I should note.

I've tried using different provides, and none seem to work. In the logs below, I had IPT, KickAss and Torrentz enabled with default settings on everything but IPT. I made sure to use the correct Username and PW on IPT as well.

I'm not really sure what else to do, as I'm very new to this. Note, in the logs, I already had Dev Mode enabled, but the logs did not seem to change between clearing and running a search again.

EDIT: So 3 seconds prior to me clicking submit new issue, I tried using TPB again in a search. I was able to get Deadpool running at a decent speed. Search gave me a torrent of quality I'd be looking for as well.. hmm.. not sure what to make of this. I will post a comment of the updated logs that include TPB in the search.

### Information:
Movie(s) I have this with: Everything, popular like Deadpool, etc.
Quality of the movie being searched: Any
Providers I use: IPTorrents
Version of CouchPotato: 3.0.1
Running on: Windows Server 2008 R2 Enterprise, Dual Xeons, etc (more than enough hardware)

### Logs:
11-09 15:57:35
INFO
[core.media.movie.searcher] Search for Deadpool in 720p ignoring ETA
11-09 15:57:35
INFO
[hpotato.core.plugins.base] Opening url: get https://kickass.unblocked.pw/m-i1431045/, data: []
11-09 15:57:35
INFO
[hpotato.core.plugins.base] Opening url: get https://torrentz.eu/feed_verified?q=%22deadpool+2016%22%20size%203000m%20-%2010000m%20seed%20%3E%200, data: []
11-09 15:57:35
ERROR
[edia._base.providers.base] Invalid XML returned, check "https://torrentz.eu/feed_verified?q=%22deadpool+2016%22 size 3000m - 10000m seed > 0" manually for issues
11-09 15:57:35
INFO
[core.media.movie.searcher] Search for Deadpool in 1080p ignoring ETA
11-09 15:57:35
INFO
[hpotato.core.plugins.base] Opening url: get https://torrentz.eu/feed_verified?q=%22deadpool+2016%22%20size%204000m%20-%2020000m%20seed%20%3E%200, data: []
11-09 15:57:36
ERROR
[edia._base.providers.base] Invalid XML returned, check "https://torrentz.eu/feed_verified?q=%22deadpool+2016%22 size 4000m - 20000m seed > 0" manually for issues

**Comment by CouchPotatoBot** [comment]
Thanks for the issue report! Before a real human comes by, please make sure your report has all the below criteria checked

- [x] Fill in all the dots. They're there for a reason
- [ ] Post logs, either inline (for smaller logs) or using [Pastebin](http://pastebin.com/)
- [ ] Post full version information

Please make sure you also read [contribution guide](https://github.com/RuudBurger/CouchPotatoServer/blob/develop/contributing.md#issues) and followed all the steps. 
Make the title describe your issue. Having "CP not working" or "I get this bug" for 100 issues, isn't really helpful. My master will close issues if there isn't enough information. On a good day he will tag the issue on close with the reason (like `can't reproduce`), but usually he won't, the lazy asshat.

Sometimes my master seems like a grumpy cat and responds with short answers. This isn't (always) because he hates you, but because he's on mobile or busy fixing bugs. If something isn't clear, please let him know. Maybe he can teach me his awesome ways.

Thanks!

**Comment by soulvoid86** [comment]
## Updated logs as stated above ^

[core.media.movie.searcher] Search for Deadpool in 720p ignoring ETA
11-09 15:57:35
INFO
[hpotato.core.plugins.base] Opening url: get https://kickass.unblocked.pw/m-i1431045/, data: []
11-09 15:57:35
INFO
[hpotato.core.plugins.base] Opening url: get https://torrentz.eu/feed_verified?q=%22deadpool+2016%22%20size%203000m%20-%2010000m%20seed%20%3E%200, data: []
11-09 15:57:35
ERROR
[edia._base.providers.base] Invalid XML returned, check "https://torrentz.eu/feed_verified?q=%22deadpool+2016%22 size 3000m - 10000m seed > 0" manually for issues
11-09 15:57:35
INFO
[core.media.movie.searcher] Search for Deadpool in 1080p ignoring ETA
11-09 15:57:35
INFO
[hpotato.core.plugins.base] Opening url: get https://torrentz.eu/feed_verified?q=%22deadpool+2016%22%20size%204000m%20-%2020000m%20seed%20%3E%200, data: []
11-09 15:57:36
ERROR
[edia._base.providers.base] Invalid XML returned, check "https://torrentz.eu/feed_verified?q=%22deadpool+2016%22 size 4000m - 20000m seed > 0" manually for issues
11-09 16:12:05
INFO
[hpotato.core.plugins.base] Opening url: get https://api.themoviedb.org/3/movie/tt1431045?api_key=xxx&append_to_response=alternative_titles, data: []
11-09 16:12:05
INFO
[hpotato.core.plugins.base] Opening url: get https://api.couchpota.to/info/tt1431045/, data: []
11-09 16:12:05
INFO
[hpotato.core.plugins.base] Opening url: get http://www.omdbapi.com/?type=movie&i=tt1431045, data: []
11-09 16:12:06
INFO
[ie.providers.info.omdbapi] Found: Deadpool (2016)
11-09 16:12:06
INFO
[hpotato.core.plugins.base] Opening url: get https://thepiratebay.mn, data: []
11-09 16:12:08
INFO
[hpotato.core.plugins.base] Opening url: get https://thepiratebay.gd, data: []
11-09 16:12:11
INFO
[hpotato.core.plugins.base] Opening url: get https://thepiratebay.la, data: []
11-09 16:12:13
INFO
[hpotato.core.plugins.base] Opening url: get https://pirateproxy.sx, data: []
11-09 16:12:14
INFO
[hpotato.core.plugins.base] Opening url: get https://api.couchpota.to/eta/tt1431045/, data: []
11-09 16:12:14
INFO
[core.media.movie.searcher] Search for Deadpool in 720p ignoring ETA
11-09 16:12:14
INFO
[hpotato.core.plugins.base] Opening url: get https://iptorrents.eu/t?l48=&q=%22deadpool%22+2016&qf=#torrents&p=1, data: []
11-09 16:12:15
INFO
[hpotato.core.plugins.base] Opening url: get https://pirateproxy.sx/search/%22Deadpool+2016%22/0/7/207, data: []
11-09 16:12:15
INFO
[hpotato.core.plugins.base] Opening url: get https://torrentz.eu/feed_verified?q=%22deadpool+2016%22%20size%203000m%20-%2010000m%20seed%20%3E%200, data: []
11-09 16:12:15
INFO
[hpotato.core.plugins.base] Opening url: get https://kickass.unblocked.pw/m-i1431045/, data: []
11-09 16:12:15
ERROR
[edia._base.providers.base] Invalid XML returned, check "https://torrentz.eu/feed_verified?q=%22deadpool+2016%22 size 3000m - 10000m seed > 0" manually for issues
11-09 16:12:17
INFO
[hpotato.core.plugins.base] Opening url: get https://pirateproxy.sx/torrent/15868499/Deadpool.2016.4K.2160p.UHD.BluRay.Hindi.Tamil.Telugu.Eng.AC3.x26, data: []
11-09 16:12:18
INFO
[edia._base.providers.base] Found: score(788) on ThePirateBay: Deadpool.2016.4K.2160p.UHD.BluRay.Hindi.Tamil.Telugu.Eng.AC3.x26
11-09 16:12:18
INFO
[hpotato.core.plugins.base] Opening url: get https://pirateproxy.sx/torrent/14809112/Deadpool_2016_(2160p_4K_Bluray_x265_HEVC_10bit_AAC_7.1_Joy)[UTR], data: []
11-09 16:12:19
INFO
[edia._base.providers.base] Found: score(599) on ThePirateBay: Deadpool 2016 (2160p 4K Bluray x265 HEVC 10bit AAC 7.1 Joy)[UTR]
11-09 16:12:20
INFO
[hpotato.core.plugins.base] Opening url: get https://pirateproxy.sx/torrent/14512204/_________Deadpool_(2016)_[HEVC.720p]_iTunes, data: []
11-09 16:12:21
INFO
[edia._base.providers.base] Found: score(416) on ThePirateBay: ?????? / Deadpool (2016) [HEVC.720p] iTunes
11-09 16:12:21
INFO
[hpotato.core.plugins.base] Opening url: get https://pirateproxy.sx/torrent/15086474/Deadpool.2016.720p.BluRay.x264-SPARKS[EtHD], data: []
11-09 16:12:22
INFO
[edia._base.providers.base] Found: score(416) on ThePirateBay: Deadpool.2016.720p.BluRay.x264-SPARKS[EtHD]
11-09 16:12:22
INFO
[hpotato.core.plugins.base] Opening url: get https://pirateproxy.sx/torrent/14690176/Deadpool.2016.BluRay.720p.DTS.x264-ETRG, data: []
11-09 16:12:23
INFO
[hpotato.core.plugins.base] Opening url: get https://api.couchpota.to/validate/ZGVhZHBvb2wuMjAxNi5ibHVyYXkuNzIwcC5kdHMueDI2NC1ldHJn/, data: []
11-09 16:12:23
INFO
[edia._base.providers.base] Found: score(387) on ThePirateBay: Deadpool.2016.BluRay.720p.DTS.x264-ETRG
11-09 16:12:23
INFO
[hpotato.core.plugins.base] Opening url: get https://pirateproxy.sx/search/%22Deadpool+2016%22/1/7/207, data: []
11-09 16:12:25
INFO
[hpotato.core.plugins.base] Opening url: get https://pirateproxy.sx/torrent/15470664/DeadPool_2016_2160p_(4K)_5.1_x264_Dual, data: []
11-09 16:12:26
INFO
[edia._base.providers.base] Found: score(179) on ThePirateBay: DeadPool 2016 2160p (4K) 5.1 x264 Dual
11-09 16:12:26
INFO
[hpotato.core.plugins.base] Opening url: get https://pirateproxy.sx/torrent/15088346/Deadpool.2016.720p.ENG-SPA.DUAL.BluRay.H264.AC3-EShare, data: []
11-09 16:12:27
INFO
[hpotato.core.plugins.base] Opening url: get https://api.couchpota.to/validate/ZGVhZHBvb2wuMjAxNi43MjBwLmVuZy1zcGEuZHVhbC5ibHVyYXkuaDI2NC5hYzMtZXNoYXJl/, data: []
11-09 16:12:27
INFO
[edia._base.providers.base] Found: score(264) on ThePirateBay: Deadpool.2016.720p.ENG-SPA.DUAL.BluRay.H264.AC3-EShare
11-09 16:12:27
INFO
[hpotato.core.plugins.base] Opening url: get https://pirateproxy.sx/torrent/14448837/Deadpool.2016.720p.BluRay.x264-SPARKS, data: []
11-09 16:12:29
INFO
[hpotato.core.plugins.base] Opening url: get https://api.couchpota.to/validate/ZGVhZHBvb2wuMjAxNi43MjBwLmJsdXJheS54MjY0LXNwYXJrcw==/, data: []
11-09 16:12:29
INFO
[edia._base.providers.base] Found: score(259) on ThePirateBay: Deadpool.2016.720p.BluRay.x264-SPARKS
11-09 16:12:29
INFO
[hpotato.core.plugins.base] Opening url: get https://pirateproxy.sx/torrent/14443968/Deadpool_2016_720p_BluRay_x264-SPARKS, data: []
11-09 16:12:30
INFO
[hpotato.core.plugins.base] Opening url: get https://api.couchpota.to/validate/ZGVhZHBvb2wgMjAxNiA3MjBwIGJsdXJheSB4MjY0LXNwYXJrcw==/, data: []
11-09 16:12:30
INFO
[edia._base.providers.base] Found: score(245) on ThePirateBay: Deadpool 2016 720p BluRay x264-SPARKS
11-09 16:12:31
INFO
[hpotato.core.plugins.base] Opening url: get https://pirateproxy.sx/search/%22Deadpool+2016%22/2/7/207, data: []
11-09 16:12:33
INFO
[hpotato.core.plugins.base] Opening url: get https://pirateproxy.sx/torrent/14735160/Deadpool_2016_BluRay_4K_2160p_5.1CH_x264_DUAL_AUDIO-Pirates82, data: []
11-09 16:12:34
INFO
[hpotato.core.plugins.base] Opening url: get https://api.couchpota.to/validate/ZGVhZHBvb2wgMjAxNiBibHVyYXkgNGsgMjE2MHAgNS4xY2ggeDI2NCBkdWFsIGF1ZGlvLXBpcmF0ZXM4Mg==/, data: []
11-09 16:12:34
INFO
[edia._base.providers.base] Found: score(128) on ThePirateBay: Deadpool 2016 BluRay 4K 2160p 5.1CH x264 DUAL AUDIO-Pirates82
11-09 16:12:34
INFO
[hpotato.core.plugins.base] Opening url: get https://pirateproxy.sx/search/%22Deadpool+2016%22/3/7/207, data: []
11-09 16:12:35
INFO
[hpotato.core.plugins.base] Opening url: get https://pirateproxy.sx/torrent/14456031/Deadpool.2016.BluRay.720p[DarkAngel], data: []
11-09 16:12:37
INFO
[edia._base.providers.base] Found: score(270) on ThePirateBay: Deadpool.2016.BluRay.720p[DarkAngel]
11-09 16:12:37
INFO
[tato.core.plugins.release] Ignored, not enough seeders, has 0 needs 1: Deadpool.2016.BluRay.720p[DarkAngel]
11-09 16:12:37
INFO
[core.downloaders.utorrent] Updated uTorrent settings to set a torrent to complete after it the seeding requirements are met.
11-09 16:12:37
INFO
[tato.core.plugins.release] Snatched "Deadpool.2016.4K.2160p.UHD.BluRay.Hindi.Tamil.Telugu.Eng.AC3.x26": Deadpool (2016) in 720p from ThePirateBay
11-09 16:12:37
INFO
[tato.core.plugins.release] Renamer disabled, marking media as finished: Deadpool (2016) in 720p
